providers/src/airflow/providers/fab/www/extensions/init_views.py

```diff
@@ -0,0 +1,67 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Copy of `airflow/www/extensions/init_views.py`. I kept only `init_plugins` and `init_error_handlers` methods